CONTRIBUTING.md

```diff
@@ -49,6 +49,18 @@ close your PR in order to prevent work being done on it.
 [issues]: https://github.com/rust-windowing/winit/issues
 [matrix]: https://matrix.to/#/#rust-windowing:matrix.org
 
+### Formatting
+
+Winit strictly relies on nightly `rustfmt`. A PR will not even get reviewed if
+nightly `cargo-fmt` check is not passed. If you have `rustup` installed you
+can install nightly toolchain using the following command `rustup toolchain install nightly`.
+
+Then to format the code you can run `cargo +nightly fmt`.
+
+However, if you feel uncomfortable using nightly toolchain, ensure that CI passes with the exception of formatting and explicitly indicate that your code should be formatted before merge.
```

Comment:
Wrap it at `80` chars.

Though, we should  add after rust formatting stuff in the same section, after `For more`:

When editing markdown files(.md), they must be wrapped at **80** chars.

Reply:
> When editing markdown files(.md), they must be wrapped at 80 chars.

Is this a convention for all repos or just this `winit`?

Reply:
it's just winit, but there's no convention how to wrap them anyway.

In general, you should just look around and follow what existing style is, in this case if you look into this file, you could notice that your line is way longer than everything else in this file...

Reply:
@kchibisov It might be limited to my knowledge.

CJK Markdowns never wrap when a string is supposed to be rendered into one line. Otherwise, unwanted spaces will be generated by the markdown parser.

Reply:
But we use `English`, which has spaces...

Generally whether you wrap something or not is up to you and project you use. If you don't wrap then you don't wrap, but we wrap. Also, if particular markdown file doesn't really handle it, then it seems like markdown parser issue and you can `\` iirc.